vic/drivers/classic/src/write_model_state.c

```diff
@@ -293,10 +298,19 @@ write_model_state(all_vars_struct *all_vars,
                            filep->statefile);
                 }
                 else {
-                    fprintf(filep->statefile, " %f", energy[veg][band].T[nidx]);
+                    fprintf(filep->statefile, " "ASCII_STATE_FLOAT_FMT, energy[veg][band].T[nidx]);
                 }
             }
 
+            /* Write foliage temperature */
```

Comment:
Good point! Will do. I haven't really tested for binary format - should we actually have a binary version of the sample data?